TeamCode/src/main/java/org/firstinspires/ftc/teamcode/opmodes/base/BaseOpMode.java

```diff
@@ -27,17 +29,31 @@ public void init(){
             driveMap.put(name,hardwareMap.get(DcMotor.class, name.toString()));
         }
         driveSystem = new DriveSystem(driveMap, hardwareMap.get(BNO055IMU.class, "imu"));
+
+        vuforia = setCamera(Vuforia.CameraChoice.CAM_RIGHT);
         DistanceSensor distanceSensor2;
         DistanceSensor distanceSensor3;
         ColorSensor colorSensor;
 
-    }
 
-    protected void setCamera(CameraChoice cameraChoice){
+    }
 
-        vuforia = new Vuforia(hardwareMap, cameraChoice);
+    protected Vuforia setCamera(CameraChoice cameraChoice){
+        int cameraMonitorViewId = hardwareMap.appContext.getResources().getIdentifier("cameraMonitorViewId", "id", hardwareMap.appContext.getPackageName());
+        WebcamName camName = null;
+        switch (cameraChoice) {
```

Comment:
Let's map the cameras in `init()`. Better still, for bonus points, let's map them all in a hardware class. See `HardwarePushbot` and `PushbotTeleopTank_Iterative` in the example code.

...if time allows.